src/lib/components/MyReservations.svelte

```diff
@@ -40,15 +40,15 @@
 	}
 
 	const bgColorByCategoryFrom = {
-		[ReservationCategory.pool]: 'from-pool-bg-from',
-		[ReservationCategory.openwater]: 'from-openwater-bg-from',
-		[ReservationCategory.classroom]: 'from-classroom-bg-from'
+		[ReservationCategory[ReservationCategory.pool]]: 'from-pool-bg-from',
```

Comment:
@nrxszvo if this needs string this might actually work as:

it would be better to update the type instead, in the types.ts if we need this key to be strings.
```
export enum ReservationStatus {
	pending = "pending",
	confirmed = "confirmed",
	rejected = "rejected",
}

export enum ReservationCategory {
	openwater = 'openwater',
	pool = 'pool',
	classroom = 'classroom'
}

```

Reply:
Ok it doesn't necessarily need to be a string value, but obviously whatever
it is, integer or string, the code needs to work with that value.  Possibly
it's ideal from a design perspective if the code does not have to be aware
of the underlying literal value of the enum at all.

In this case, to my surprise, it seems that even though the ReservationData
type defines the category field as an enum with integer literal values, it
doesn't cause any runtime errors if the actual reservation objects have
string values for the category field.  I would have guessed that it would,
but I still don't understand typescript too well yet.  Is this expected
behavior?  I thought that the value of declaring reservation objects with
the ReservationData type was that it would cause runtime errors in cases
like this.

Reply:
the problem was the overriding of the type coming from xata,

xata's category, status are only strings.

```
type ReservationData = xata.types & {
   category: EnumType
}
```

it was already previously configured as just string, when i overrided the type to be enum it was being saved a integer ( enum defaults to int keys in ts ).

anyway i fixed the value in the type itself and this pr is not needed anymore